fvp.mk

```diff
@@ -106,6 +149,11 @@ LINUX_DEFCONFIG_COMMON_FILES := \
 		$(LINUX_PATH)/arch/arm64/configs/defconfig \
 		$(CURDIR)/kconfigs/fvp.conf
 
+.PHONY: linux-ftpm-module
+linux-ftpm-module: linux
+	$(MAKE) -C $(LINUX_PATH) $(LINUX_COMMON_FLAGS) M=drivers/char/tpm  \
+		modules_install INSTALL_MOD_PATH=$(LINUX_PATH)
+
```

Comment:
Shouldn't the `linux-ftpm-module` target do nothing when `USE_MEASURED_BOOT` is disabled?

Reply:
Actually, an fTPM service does not need Measured Boot, so it could have been kept enabled anyway, hence I decided to leave it that way in order to not increase the complexity of this file. 

However, after thinking a bit more about it, it might not make sense to have the fTPM enabled without measured boot in this context so I made the build conditional as well as other ancillary components used to access the fTPM service. It was a couple of lines anyway.